server/neptune/workflows/activities/github/commit.go

```diff
@@ -0,0 +1,6 @@
+package github
+
+type Commit struct {
+	Revision string
+	Branch   string
+}
```

Comment:
I think you should just add Branch as a separate request parameter so that we don't have to worry about request serialization breaking across gateway and our workers.